conda_build/variants.py

```diff
@@ -634,6 +634,14 @@ def get_vars(variants, loop_only=False):
     return loop_vars
 
 
+def is_loop_var(var, variants):
+    if var in {'pin_run_as_build', 'zip_keys', 'ignore_version'}:
```

Comment:
Some questions: where is this set of vars come from, can we set it as a global variable and add some documentation so it becomes easier to extend if needed later on?

Reply:
I added it as a global variable. The original usage comes from `get_vars` function above, so I'm not sure if my understanding is correct.